admins/pageflow/user.rb

```diff
@@ -122,9 +122,11 @@ module Pageflow
 
     collection_action :quota_state do
       @account = Pageflow::Account.find(params[:account_id])
-      authorize!(:add_member_to, @account)
-
-      render(layout: false)
+      if can?(:see_user_quota, @account)
```

Comment:
ActiveAdmin supports multiple [authorization adapters](https://activeadmin.info/13-authorization-adapter.html#using-the-cancan-adapter) and provides a more general `authorized?` helper that will work with any of them. Using `can?` here has the same effect but depends on the fact that we use the CanCan adapter.